x-pack/legacy/plugins/ingest/docs/spec/asset_install-001.md

```diff
@@ -0,0 +1,55 @@
+# Asset Install APIs
+Asset instilation APIS will be JS/TS only as they are intended for use only by plugins within Kibana and not a user-facing system though there is the ability to expose this to a REST interface in the future should we decide that is needed.
+
+## Asset types
+The following asset types have support planned:
+
+**ES Assets**
+- Ingest pipelines
+- Index Templates
+- ILM Policies
+- Rollup Jobs
+- ML Jobs*
```

Comment:
I would remove it from this doc for now. Removing ML jobs is a bit tricky. Later on we definitively must support it.